_posts/2020-07-09-implementing-retry-with-resilience4j.md

```diff
@@ -154,7 +154,7 @@ Supplier<List<Flight>> flightSearchSupplier = () -> {
 };
 ```
 
-So what should we do when our remote call can throw an `Exception`? We can use the `Retry.decorateCheckedSupplier()` (or the `executeCheckedSupplier()` instance method) instead of `Retry.decorateSupplier()`:
+So what should we do when we want to retry for all exceptions that our remote call can throw? We can use the `Retry.decorateCheckedSupplier()` (or the `executeCheckedSupplier()` instance method) instead of `Retry.decorateSupplier()`:
```

Comment:
Again, it says checked exceptions are being retried

Perhaps move everything about checked exception into a section "Working with Checked Exceptions"? It's not "Simple Retry" anymore, after all, as this discussion shows :)

Reply:
Please see my comment above, checked exceptions *are* retried. Not sure when you played around, but please take latest code and run the basicExample_ServiceThrowingException_v2() method.

Agree on the sections, I have moved it into a separate subsection.  :)

Reply:
Ok, got it now, thanks for clearing that up. I must have looked at a different method in the example code.